### PR TITLE
Pub prep

### PR DESCRIPTION
**Pull request by amilan17**
DO NOT MERGE!! 

Many refactors for WIS2 manual, change caps, embed URLs, reformat requirements class, etc. Some editorial changes to consider adding into main branch.

### CODE REVIEW

**Comment by amilan17** [comment]
@tomkralidis

File: standard/requirements/core/REQ_extent_geospatial.adoc [hunk 1]
```diff
@@ -2,7 +2,7 @@
 [width="90%",cols="2,6a"]
 |===
 ^|*Requirement {counter:req-id}* |*/req/core/extent_geospatial*
-^|A |A WCMP record SHALL provide ONE `+geometry+` property to convey the geospatial properties of a dataset using a geographic coordinate reference system (World Geodetic System 1984 [WGS 84]) and longitude and latitude decimal degree units.
-^|B |The `+geometry+` values SHALL be integer or float data types.
-^|C |The `+geometry+` property SHALL provide the value of `+null+` when geometry cannot be derived.
+^|A |A WCMP record shall provide ONE `+geometry+` property to convey the geospatial properties of a dataset using a geographic coordinate reference system (World Geodetic System 1984 [WGS 84]) and longitude and latitude decimal degree units.
+^|B |The `+geometry+` coordinates shall be integer or float data types.
```
[the remark this answers is not in the document]
Comment (tomkralidis) [reply standard/requirements/core/REQ_extent_geospatial.adoc:6]: already in main.

File: standard/recommendations/core/REC_extent_geospatial_precision.adoc [hunk 2]
```diff
@@ -2,5 +2,5 @@
 [width="90%",cols="2,6a"]
 |===
 ^|*Recommendation {counter:rec-id}* |*/rec/core/extent_geospatial_precision*
-^|A |Geometry coordinates SHOULD have a level of precision of 2 decimal places.
+^|A |Geometry coordinates should have a level of precision of at least 2 or more decimal places.
 |===
```
[the remark this answers is not in the document]
Comment (tomkralidis) [reply standard/recommendations/core/REC_extent_geospatial_precision.adoc:6]: Will update.

File: standard/requirements/core/REQ_identifier.adoc [hunk 3]
```diff
@@ -2,9 +2,9 @@
...
 |===
 ^|*Requirement {counter:req-id}* |*/req/core/identifier*
-^|A |A WCMP record SHALL provide an identifier via the `+id+` property.
-^|B |The `+id+` property SHALL have the following notation: `+urn:x-wmo:md:{centre_id}:{local_identifier}+`.
-^|C |The ``centre_id`` SHALL be based on the associated vocabulary specified in the WIS <<wis2-topic-hierarchy, topic hierarchy>>.
-^|D |The `+id+` property SHALL include a local identifier as defined by the data publisher.  The local identifier SHALL NOT have spaces or special or accented characters.
+^|A |A WCMP record shall provide an identifier via the `+id+` property.
+^|B |The `+id+` property shall have the following notation: `+urn:x-wmo:md:{centre_id}:{local_identifier}+`.
+^|C |The ``centre_id`` shall be based on the associated vocabulary specified in the WIS2 Topic Hierarchy (see Appendix E).
+^|D |The `+id+` property shall include a local identifier as defined by the data publisher.  The local identifier shall not have spaces or accented characters.
```
[the remark this answers is not in the document]
Comment (tomkralidis) [reply standard/requirements/core/REQ_identifier.adoc]: TBD/pending #182

File: standard/sections/clause_7_normative_text.adoc [hunk 4]
```diff
@@ -368,14 +366,10 @@ The `+additionalExtents.spatial+` property is for other geospatial extents assoc
...
-signed number between -180 and 180, greater than or equal to minx.
-* maxy is the northernmost coordinate of the limit of the dataset extent, expressed in latitude decimal degrees as a
-signed number between -90 and 90, greater than or equal to miny.
+* The minimum longitude is the westernmost coordinate of the limit of the dataset extent, expressed in longitude decimal degrees as a signed number between ``-180`` and ``180``, less than or equal to maximum longitude.
+* The minumum latitude is the southernmost coordinate of the limit of the dataset extent, expressed in latitude decimal degrees as a signed number between ``-90`` and ``90``, less than or equal to maximum latitude.
+* The maximum longitude is the easternmost coordinate of the limit of the dataset extent, expressed in longitude decimal degrees as a signed number between ``-180`` and ``180``, greater than or equal to minimum longitude.
+* The maximum latitude is the northernmost coordinate of the limit of the dataset extent, expressed in latitude decimal degrees as a signed number between ``-90`` and ``90``, greater than or equal to minimum latitude.
```
[the remark this answers is not in the document]
Comment (tomkralidis) [reply standard/sections/clause_7_normative_text.adoc:372]: Will apply to main.

File: standard/sections/clause_7_normative_text.adoc [hunk 5]
```diff
@@ -481,7 +475,7 @@ If necessary, further granularity can be indicated by the `+additionalExtents.te
 ...
 "properties": {
   ...
-  "description": "ICON-EPS 0.5\u00b0 x 0.5\u00b0 regular lat/lon grid, up to +180h every 6h, runs 00/12 UTC, various parameter, various level, various threshold",
+  "description": "ICON-EPS 0.5 deg x 0.5 deg regular lat/lon grid, up to +180h every 6h, runs 00/12 UTC, various parameter, various level, various threshold",
```
[the remark this answers is not in the document]
Comment (tomkralidis) [reply standard/sections/clause_7_normative_text.adoc:478]: Will apply.

**Pull request closed without merging**